core-plugins/docs/XMLReader-batchsource.md

```diff
@@ -37,6 +37,9 @@ will not be processed again on future runs of the pipeline.
 
 **Table Data Expiry Period (Days):** The amount of time (in days) to wait before clearing the table used to track processed filed. If omitted, data will not expire in the tracking table. Example: for `tableExpiryPeriod = 30`, data before 30 days is deleted from the table.
 
+**Enable external entities:** This enables processing external entities while reading xml file. Defaults to `false`.
```

Comment:
nit: Can we add a warning here that external entities and DTD support should be disabled unless absolutely necessary for security risks? That way users can be aware of the risk of using these options. Thanks!